### ISSUE TITLE
Uncaught ReferenceError: jQuery is not defined (fixDW-libs.js)

### ISSUE THREAD

**Issue by illia108** [mihai-vlc/bm-extender#1]
Hi. I get these errors when going to a custom controller page:
```
Uncaught ReferenceError: jQuery is not defined
    at fixDW-libs.js:22
    at fixDW-libs.js:22
(anonymous) @ fixDW-libs.js:22
(anonymous) @ fixDW-libs.js:22
fixDW.js:482 Uncaught ReferenceError: jQuery is not defined
    at fixDW.js:482
```

**Comment by mihai-vlc** [comment]
this should be fixed in 1.3.3

**Comment by illia108** [comment]
Yes. No errors. Thank you!
